src/main/java/com/crud/tasks/domain/TrelloListDto.java

```diff
@@ -0,0 +1,21 @@
+package com.crud.tasks.domain;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+
+@AllArgsConstructor
+@Getter
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class TrelloListDto {
+
+    @JsonProperty("id")
+    private String id;
+
+    @JsonProperty("name")
+    private String name;
+
+    @JsonProperty("isClosed")
+    private boolean isClosed;
```

Comment:
przy mapowaniu danych stosuj zawsze typy obiektowy. Obecnie kiedy zmapujesz obiekt i isClosed bedzie false, to tak naprawde nie wiesz czy json zawieral informacje false, czy nie bylo w ogole tej informacji (boolean jako typ prosty ma zawsze domyslna wartosc false)